src/net/contingency.c

```diff
@@ -277,7 +278,7 @@ char* CONT_get_json_string(Cont* cont) {
     return NULL;
 
   // Output
-  output = (char*)malloc(sizeof(char)*CONT_BUFFER_SIZE*10);
+  output = (char*)malloc(sizeof(char)*buffer_size*5);
```

Comment:
Can we use here CONT_JSON_BUFFER_SIZE*10 and avoid introducing "buffer_size" and not use "JSON_STR_BUFFER_EXTRA"?